functions/go/set-namespace/transformer/namespace.go

```diff
@@ -31,18 +31,23 @@ func SetNamespace(rl *fn.ResourceList) (bool, error) {
 		return true, nil
 	}
 	// Update "namespace" to the proper resources.
-	tc.Transform(rl.Items)
+	count := tc.Transform(rl.Items)
```

Comment:
Note that some of our functions in the catalog tracks not only `count` but also each field that is updated. e.g. https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/set-image/v0.1/examples/set-image-simple/.expected/results.yaml

Reply:
Good point. The `count` comes from the fishfooding issue b/228617774. I will add the function specification result as well.

Reply:
Actually, adding the result info in resource level might end up output too much details. created https://github.com/GoogleContainerTools/kpt/issues/2996 to track further discussions. I would prefer keeping the output short as an update summary.

Reply:
There is no knob in the function spec that controls the output of a function.
IMO a function should provide as much information as possible, but the function orchestrator (kpt CLI in this case) should decide how verbose the output should be. (e.g. hide `info` and `warning` at different verbosity level).

It's not a hard requirement for this PR, but a nice-to-have feature.

Reply:
Right, so I propose adding a new kpt flag to control the output severity level, for both `kpt fn eval` and `kpt fn render`. https://github.com/GoogleContainerTools/kpt/issues/2996
But one thing I don't like is that `kpt fn` has already had too many flags. kustomize sets a preference to configure the result in kustomization.yaml. But I don't think that's applied to the severity either. Do you have any suggestions how we can configure that?